pkg/cluster/nodes/node.go

```
@@ -273,5 +281,23 @@ func getProxyDetails() proxyDetails {
 			}
 		}
 	}
+
+	// Specifically add the docker network subnets to NO_PROXY
+	subnets, err := getSubnets(defaultNetwork)
+	if err == nil {
```

is ok to ignore the error? just curiosity

---

I previously had it log the error, but decided to get rid of it since there's not really anything to do if we fail to get the subnets.

---

it should probably fail?

---

not really? everything else would work just fine for users that aren't using a proxy.

---

if we didn't detect any proxy info we can elide even querying this instead